amundsen_application/static/js/components/DashboardPage/index.tsx

```diff
@@ -154,6 +154,7 @@ export class DashboardPage extends React.Component<DashboardPageProps, Dashboard
             <BookmarkIcon bookmarkKey={ dashboard.uri } resourceType={ ResourceType.dashboard } />
             <div className="body-2">
               Dashboard in&nbsp;
+              { indexDashboardsIsBeta() && <Flag text="beta" labelStyle="default"/> }
```

Comment:
Given that this code is temporary because of our design needs as opposed to an official Amundsen feature, do you still feel the need to set of these configurations vs. hardcoding the `Flag`? I don't expect to create an official release until we finish our beta phase, at which point all this code would be removed. So I'd be fine with introducing the least amount of code.

Reply:
Will hard code.